src/data_structure/versions/add_min_capacity_margin.jl

```diff
@@ -0,0 +1 @@
+# Empty migration script - template update is all that is required
```

Comment:
I think this is a deal-breaker at this point. We need to make it work more robustly because one can get very very lost.

Reply:
@manuelma I must have understood - but what is your idea for handling migrations that only need a template update? A template update will only be triggered if the version increases and the version is the number of upgrade scripts, so what is the alternative? If this can be done quicky, then that's great - otherwise, it's holding up all development?

Reply:
Maybe you're right, maybe we can use the empty migration script but two things:

- We don't need a separate file and function per migration. Just write a single funtion in migrations.jl called "_do_nothing" or something like that that returns true, and insert that function as many times as needed and in as many positions as needed within the _upgrade_functions Array.
- It doesn't work if the user runs with upgrade=false and that will be a headache for our users that pull the master until we do something. We can keep developing and helping those users as they come, but I would like to do something rather soon.

Reply:
Actually, run_migrations imports the template at the end regardless of the version. Why did you find that you needed to bump the version in order to load the template?

Did you try running with upgrade=true and without bumping the version in the template?